.github/workflows/push-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Run tests on push
+on:
+  push:
+    branches:
+      - master
+      - f[0-9]+-devel
+      - f[0-9]+-release
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Build anaconda-ci container
+        run: make -f Makefile.am anaconda-ci-build
```

Comment:
Do not build the container, just let it download from our quay.io repository. These will be downloaded automatically on when starting the tests. So just removing these two lines should be enough.

Or re-use this part https://github.com/rhinstaller/anaconda/blob/master/.github/workflows/tests.yml#L57 .